Composer/packages/extensions/adaptive-form/src/components/fields/RecognizerField.tsx

```diff
@@ -37,11 +63,25 @@ const RecognizerField: React.FC<FieldProps<MicrosoftIRecognizer>> = (props) => {
       return;
     }
 
+    selected.map((s) => {
+      if (s === SDKKinds.LuisRecognizer) {
+        onChange(`${currentDialog.id}.lu.qna`);
+      }
+    });
+    if (selected[0] === SDKKinds.LuisRecognizer) {
+      selected[0] = SDKKinds.CrossTrainedRecognizerSet;
+    }
```

Comment:
make this section more descriptive, hard to understand when and why the kind should be transformed.

Reply:
Add comment. This is because there is only cross trained recognizer in the dropdown options. So we should transform it for old bot